api/AuthenticatorAssertionResponse.json

```
@@ -16,7 +16,15 @@
             ]
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": "65",
+            "notes": "Only support USB U2F tokens",
```

This entry (and all the other instances) reads a little weird, maybe ungrammatical, at least to me. How about this?

```suggestion
            "notes": "Only USB U2F tokens are supported.",
```